include/torch-mlir/Dialect/Torch/Transforms/Passes.h

```diff
@@ -57,6 +62,10 @@ struct TorchLoweringPipelineOptions
   ListOption<std::string> backendLegalOps{
       *this, "backend-legal-ops",
       llvm::cl::desc("List of ops to be considered legal for the backend.")};
```

Comment:
With the change I suggested below, we should update this description to be "List of ops to be considered legal for the backend, such as 'aten.foo'"). (I think the description might be duplicated elsewhere too).